build-info-extractor-maven3/src/test/java/org/jfrog/build/extractor/maven/resolver/ArtifactoryResolutionTest.java

```diff
@@ -0,0 +1,163 @@
+package org.jfrog.build.extractor.maven.resolver;
+
+import org.eclipse.aether.repository.RemoteRepository;
+import org.eclipse.aether.repository.RepositoryPolicy;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+@Test
+public class ArtifactoryResolutionTest {
+
+    String releaseUrl = "www.release-url.com";
+    String httpReleaseUrl = "http://" + releaseUrl;
```

Comment:
Please don't leave compiler warnings in the code
```suggestion
    @SuppressWarnings("HttpUrlsUsage")
    String httpReleaseUrl = "http://" + releaseUrl;
```